Lib/argparse.py

```diff
@@ -1042,7 +1042,7 @@ def __init__(self,
                  version=None,
                  dest=SUPPRESS,
                  default=SUPPRESS,
-                 help="show program's version number and exit"):
+                 help=_("show program's version number and exit")):
```

Comment:
Hi!

This change will not work if the `gettext` global domain is set (via `gettext.textdomain()`, for instance) after the `argparse` module is imported: the translation here happens as soon as `argparse` is imported.

I would suggest the following change instead, so that the string is translated only when actually creating an  instance of `_VersionAction`:
```suggestion
                 help=None):
        if help is None:
            help = _("show program's version number and exit")
```

Thanks!

Kind regards,
Jérémie.